polyfills/IntersectionObserver/polyfill.js

```diff
@@ -48,11 +41,7 @@ function IntersectionObserverEntry(entry) {
   this.rootBounds = entry.rootBounds;
   this.boundingClientRect = entry.boundingClientRect;
   this.intersectionRect = entry.intersectionRect || getEmptyRect();
-  try {
-    this.isIntersecting = !!entry.intersectionRect;
-  } catch (err) {
-    // This means we are using the IntersectionObserverEntry polyfill which has only defined a getter
```

Comment:
We need this part because we the IntersectionObserverEntry polyfill is only a getter https://github.com/Financial-Times/polyfill-service/blob/master/polyfills/IntersectionObserverEntry/polyfill.js

Reply:
Ok I can add it back

Reply:
The blocking review is still here even with the rollback oO 
https://github.com/Financial-Times/polyfill-service/pull/1387/commits/d4eac9e05256110f93cbee66d150a2e32a3897c1